mypy.ini

```diff
@@ -10,3 +10,11 @@ strict_concatenate = True
 disallow_subclassing_any = True
 disallow_untyped_decorators = True
 disallow_any_generics = True
+exclude=/tomllib/
+mypy_path=.pixi/env/Library/python
```

Comment:
Probably good for @evetion to check if that path also works for unix-like systems

Reply:
For me its under `.pixi/env/share/qgis/python` (assuming you expect a `qgis` folder in there with Python files).

Reply:
Ha, didn't know they were so different. I guess we could just list both? `mypy_path=.pixi/env/Library/python:.pixi/env/share/qgis/python`